### PR TITLE
错别字修改

### PR DESCRIPTION
**Pull request by professordeng**

### CODE REVIEW

File: content/why_rxswift.md [hunk 1]
```diff
@@ -249,5 +249,5 @@ Observable.zip(
 * 复合 - Rx 就是复合的代名词
 * 复用 - 因为它易复合
 * 清晰 - 因为声明都是不可变更的
-* 易用 - 因为它抽象的了异步编程，使我们统一了代码风格
+* 易用 - 因为它抽象了的异步编程，使我们统一了代码风格
```
Comment (beeth0ven) [inline content/why_rxswift.md]: If it's better to remove '的' as '因为它抽象了异步编程'?

**Review by beeth0ven — COMMENTED** [review]
If it's better to remove '的'

**Review by professordeng — COMMENTED** [review]
谢谢指正，已修改～

**Comment by beeth0ven** [comment]
Thank you!

**Pull request merged**